From ca50cb058eeefe30442baf392af72bd778cb6e20 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 30 Sep 2019 18:00:16 +0800
Subject: [PATCH] ASoC: rockchip: pdm: Fix compile warning

sound/soc/rockchip/rockchip_pdm.c: In function 'rockchip_pdm_hw_params':
sound/soc/rockchip/rockchip_pdm.c:179:3: warning: 'clk_out' may be used uninitialized in this function [-Wmaybe-uninitialized]

Change-Id: I10b240781043a61ce365e6349a102b625db433c7
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 sound/soc/rockchip/rockchip_pdm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/rockchip/rockchip_pdm.c b/sound/soc/rockchip/rockchip_pdm.c
index ce0e68900f60..a98e4f04e11f 100644
--- a/sound/soc/rockchip/rockchip_pdm.c
+++ b/sound/soc/rockchip/rockchip_pdm.c
@@ -158,7 +158,7 @@ static int rockchip_pdm_hw_params(struct snd_pcm_substream *substream,
 	struct rk_pdm_dev *pdm = to_info(dai);
 	unsigned int val = 0;
 	unsigned int clk_rate, clk_div, samplerate;
-	unsigned int clk_src, clk_out;
+	unsigned int clk_src, clk_out = 0;
 	unsigned long m, n;
 	bool change;
 	int ret;
-- 
2.35.3

